src/constants.ts

```diff
@@ -11,6 +11,7 @@ let ENVIRONMENT: string;
 let SENTRY_DSN: string;
 
 const GENEZIO_TELEMETRY_ENDPOINT = "https://c4h2bia7gbokqdxxc6fe5sgj5e0imchy.lambda-url.us-east-1.on.aws/";
+const GENEZIO_REGISTRY = "5lraaxyjfn2jjgroseiwuzn3h40hmjxf.lambda-url.us-east-1.on.aws/RegistryHTTPHandler";
```

Comment:
is this the same one for both dev and prod?

Reply:
initially i thought so, but i think we need to specify another one for prod. will leave as a placeholder for now